lms/djangoapps/grades/tests/integration/test_events.py

```diff
@@ -28,8 +27,10 @@ class GradesEventIntegrationTest(ProblemSubmissionTestMixin, SharedModuleStoreTe
     of the grading infrastructure.
     """
     @classmethod
-    def setUpClass(cls):
-        super(GradesEventIntegrationTest, cls).setUpClass()
+    def reset_course(cls):
+        """
+        Sets up the course anew.
+        """
         with cls.store.default_store(ModuleStoreEnum.Type.split):
             cls.course = CourseFactory.create()
             cls.chapter = ItemFactory.create(
```

Comment:
[This comment explained it, but got collapsed :(](https://github.com/edx/edx-platform/pull/14529#discussion_r101403690)

Basically, if the second test gets run first, it breaks the course and the first test fails.